Patch-Source: https://sources.debian.org/src/rnp/0.15.2-6/debian/patches/0002-Avoid-testing-chunk-sizes-greater-than-16.patch

From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Fri, 29 Oct 2021 15:53:27 -0400
Subject: Avoid testing chunk sizes greater than 16

In draft-ietf-openpgp-crypto-refresh-04, chunk size is capped at 16.
We don't need to test more than that, and it looks like tests of chunk
size 30 fail on 32-bit platforms like armel and armhf (though they
don't fail on i386 for some reason i don't understand).

(see https://github.com/rnpgp/rnp/issues/1664)
---
 src/tests/cli_tests.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/tests/cli_tests.py b/src/tests/cli_tests.py
index 2a6281d..06de065 100755
--- a/src/tests/cli_tests.py
+++ b/src/tests/cli_tests.py
@@ -2060,8 +2060,7 @@ class Encryption(unittest.TestCase):
         AEAD_C = list_upto(['AES', 'AES192', 'AES256', 'TWOFISH', 'CAMELLIA128',
                             'CAMELLIA192', 'CAMELLIA256'], Encryption.RUNS)
         AEAD_M = list_upto([None, 'eax', 'ocb'], Encryption.RUNS)
-        AEAD_B = list_upto([None, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 12, 14, 18,
-                            24, 30, 40, 50, 56], Encryption.RUNS)
+        AEAD_B = list_upto([None, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 12, 14, 16], Encryption.RUNS)
 
         usegpg = gpg_supports_aead()
 
